docs/src/main/sphinx/language/sql-support.rst

```diff
@@ -143,6 +143,7 @@ Connector roles:
 
 Grants management:
 
+* :doc:`/sql/deny`
```

Comment:
we would have to check which connectors link to this section and say they support this and confirm it to be true .. or implement .. or update

Reply:
Did you check ?

Reply:
Connectors don't implement access control. But yes, none of the access controls support this. It is part of SPI to allow people to integrate with systems which have concept of DENY.